.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: CI for sbt-riffraff-artifact
+
+on:
+  pull_request:
+  # Manual invocation.
+  workflow_dispatch:
+  push:
+    branches:
+      - $default-branch
+env:
+  FINAL_TEAMCITY_BUILD: 1309
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    permissions:
+      # required by guardian/actions-assume-aws-role
+      id-token: write
+      contents: read
+    steps:
+    - name: Checkout branch
+      uses: actions/checkout@v2
+    - uses: guardian/actions-assume-aws-role@v1
+      with:
+        awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+    - name: Get node version
+      run: echo NODE_VERSION=$(cat .nvmrc) >> $GITHUB_ENV
+    - uses: actions/setup-node@v2
+      with:
+          node-version: ${{ env.NODE_VERSION }}
+    - name: Set up JDK
+      uses: actions/setup-java@v2
+      with:
+        java-version: 8
+        distribution: adopt
+    - name: CI yarn
+      shell: bash
+      run: | 
+        yarn install --force --frozen-lockfile
+        yarn lint
+        yarn test
+        yarn build
+    - name: CI for sbt-riffraff-artifact
+      shell: bash
+      run: |
+        export GITHUB_RUN_NUMBER=$(( $GITHUB_RUN_NUMBER + $FINAL_TEAMCITY_BUILD ))
+        sbt clean compile
```

Comment:
You'll want `riffRaffUpload` on the end too.

Reply:
I excluded this to avoid creating builds until I was ready to disable TC. I will add it back in shortly.